localstack/services/awslambda/lambda_executors.py

```diff
@@ -733,7 +734,7 @@ def docker_image_for_runtime(cls, runtime):
         docker_image = config.LAMBDA_CONTAINER_REGISTRY
         # TODO: remove prefix once execution issues are fixed with dotnetcore/python lambdas
         # See https://github.com/lambci/docker-lambda/pull/218
-        lambdas_to_add_prefix = ['dotnetcore', 'python2.7', 'python3.6', 'python3.7']
+        lambdas_to_add_prefix = ['dotnetcore2.0', 'dotnetcore2.1', 'python2.7', 'python3.6', 'python3.7']
```

Comment:
FYI, it looks like the GitHub issue comment above this change has been fixed/closed. You might be able to remove this prefix now (the tags do exist based on what i saw, at least for the dotnetcore images). I thought about updating that, but figured it'd be best to separate that from this specific addition.